### DISCUSSION TITLE
Skin versions

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#17376] [Ideas] [answered]
There are some skins, that has different versions for different mods (i.e. DT or EZ) that share same interface but have different gameplay elements
It would be really cool if there was an option to make this versions being stored in the same skin folder and in game there would be a drop-down menu for each skin which would allow you to chose the version
It can be implemented in the same way as it works for combo/score/hitcircle prefix in skin.ini but for other gameplay elements

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/15626
https://github.com/ppy/osu/discussions/12977

**Reply by crumpledmint** [reply]
I feel like this discussion threads not fully representing my idea, though it has some of the elements, that I told about. 
I'm talking not only about grouping skins, but also about grouped skins sharing interface, so it take less hard drive space and less time to modify skins (no more copying all of the interface elements, which takes a lot of time cause there is no way to sort all of them together without gameplay elements)

**Reply by bdach** [reply]
with the new lazer storage structure, if multiple skins use one common image, only one copy of it will be stored anyway. and it doesn't depend on filename either.
